matlab/CARFAC_SAI_hacking.m

```diff
@@ -20,6 +20,19 @@
 
 clear variables
 
+% Clear old files
```

Comment:
Use matlab's delete function instead:
delete('frames/*')
delete('frames')

Also instead of if false, I prefer this way:
clear_old_files = false
if clear_old_files

Reply:
Done